JournalMonitor/JournalMonitor.cs

```diff
@@ -1245,11 +1245,14 @@ public static List<Event> ParseJournalEntry(string line)
                                     channel == "wing" ||
                                     channel == "friend" ||
                                     channel == "voicechat" ||
-                                    channel == "local"
+                                    channel == "local" ||
+                                    channel == null
                                 )
                                 {
                                     // Give priority to player messages
                                     source = channel == "wing" ? "Wing mate" : "Commander";
+                                    source = channel == null ? "Crew mate" : "Commander";
```

Comment:
I think will overwrite the "wing" channel, won't it?
Let's add a unit test for each case.